test/pie-itemRadiusScale.html

```diff
@@ -0,0 +1,291 @@
+                 * but full radius and semitransparent background */
+                createBackgroundSeries([1, 2]),
+                createScaledSeries(
+                    [1, 2],
+                    {},
+                    itemScale
+                ),
+
+                /** example #3: add donut with the same angles,
+                 * but full outer radius and semitransparent background */
+                createBackgroundSeries(
+                    [1, 3],
+                    {radius: ['13%', '22%']}
+                ),
+                createScaledSeries(
+                    [1, 3],
+                    {radius: ['13%', '22%']},
+                    itemScale
+                ),
+
+                /** example #4: almost the same as examples #2,
+                 * but non-linear function used */
+                createBackgroundSeries([2, 1]),
+                createScaledSeries(
+                    [2, 1],
+                    {encode: ['value', 'selected']},
+                    itemScaleCubic
+                ),
+
+                /** example #5: almost the same as examples #2,
+                 * but non-linear function used */
+                createBackgroundSeries([2, 2]),
+                createScaledSeries(
+                    [2, 2],
+                    {encode: ['value', 'selected']},
+                    itemScaleQuadratic
+                ),
+
+                /** example #6: almost the same as examples #3,
+                 * but non-linear function used */
+                createBackgroundSeries(
+                    [2, 3],
+                    {radius: ['13%', '22%']}
+                ),
+                createScaledSeries(
+                    [2, 3],
+                    {
+                        radius: ['13%', '22%'],
+                        encode: ['value', 'selected']
+                    },
+                    itemScaleQuadratic
+                ),
+            ]
+        });
+
+        function createTitle(pos, text) {
+            return {
+                subtext: text,
+                textAlign: 'center',
+                left: `${(pos[1] / 3 - 1 / 6) * 100}%`,
+                top: `${(pos[0] - 1) * 50 + 6}%`
+            };
+        }
+
+        /**
+         * @param pos {number[]}
+         * @param [radius] {string[]|number[]}
+         * @param [encode] {string|string[]}
+         * @param itemRadiusScale {function}
+         * @return {*}
+         */
+        function createScaledSeries(pos, {radius, encode = 'value'} = {}, itemRadiusScale) {
+            return {
+                name: `Selected ${encode instanceof Array ? encode[0] : encode}`,
+                type: 'pie',
+                center: [
+                    `${(pos[1] / 3 - 1 / 6) * 100}%`,
+                    `${(pos[0] - 1) * 50 + 25}%`,
+                ],
+                radius: radius || '22%',
+                z: 10,
+                emphasis: {disabled: true},
```

Comment:
If `backgroundStyle` is provided, then there should be no need to disable emphasis.